service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -122,6 +124,22 @@ public void updateEhrExtractStatusCore(SendEhrExtractCoreTaskDefinition sendEhrE
         }
     }
 
+    public void updateEhrExtractStatusContinue(String conversationId) {
+        Query query = createQueryForConversationId(conversationId);
+
+        Update update = createUpdateWithUpdatedAt();
+        Instant now = Instant.now();
+        update.set(CONTINUE_RECEIVED_PATH, now);
+        UpdateResult updateResult = mongoTemplate.updateFirst(query, update, EhrExtractStatus.class);
```

Comment:
you could use findAndModify here, it will return you updated record and you will to have to query it again.